libs/feature-flags/src/lib/feature-flags.ts

```diff
@@ -1,6 +1,12 @@
 import { Client } from './configcat'
 import { FeatureFlagClient, FeatureFlagClientProps } from './types'
 
+export const sdkKeys = {
```

Comment:
I don't want these in code. They're provided in the environment :grimacing:

Reply:
:grimacing: 
same problem as in the first thread, lets have this discussion there.